applications/unit_tests/detail/instance_extensions/surface_cocoa.hpp

```diff
@@ -29,22 +29,20 @@ inline void testInstanceSurfaceCocoa()
 {
     llri::surface_cocoa_desc_ext empty {};
     
-    SUBCASE("Extension not enabled")
-    {
-        llri::instance_desc desc {};
-        llri::Instance* instance;
-        REQUIRE_EQ(llri::createInstance(desc, &instance), llri::result::Success);
-        
-        llri::SurfaceEXT* surface;
-        CHECK_EQ(instance->createSurfaceEXT(empty, &surface), llri::result::ErrorExtensionNotEnabled);
-        
-        llri::destroyInstance(instance);
-    }
+    // check with extension not enabled
+    llri::instance_desc desc {};
+    llri::Instance* instance;
+    REQUIRE_EQ(llri::createInstance(desc, &instance), llri::result::Success);
     
+    llri::SurfaceEXT* surface;
+    CHECK_EQ(instance->createSurfaceEXT(empty, &surface), llri::result::ErrorExtensionNotEnabled);
+    
+    llri::destroyInstance(instance);
+
+    // check with extension enabled
     if (llri::queryInstanceExtensionSupport(llri::instance_extension::SurfaceCocoa))
     {
         glfwInit();
-        // disable the default OpenGL context that GLFW creates
```

Comment:
any particular reason for removing the comment?

Reply:
comment was there because the code was copied over from a sample, just seems a bit repetitive to keep repeating it here